src/skeletons/styles.tsx

```diff
@@ -0,0 +1,29 @@
+import {createUseStyles} from '../jss';
+
```

Comment:
This question is for @atabel 

In other components we have "base" component, should we have something similar with skeletons? skeleton-base.tsx

For example, textField use text-field-base, something similar for feedbacks, successFeedbackScreen use FeedbackScreen as a base.

Reply:
those are internal implementation details, not a big deal